src/redist/redist.csproj

```diff
@@ -264,4 +264,18 @@
     <Copy SourceFiles="@(Stage2Cli)"
           DestinationFiles="@(Stage2Cli->'$(Stage2WithBackwardsCompatibleRuntimesOutputDirectory)/%(RecursiveDir)%(Filename)%(Extension)')" />
   </Target>
+
+  <Target Name="RetargetVSTestConsole"
+          AfterTargets="Publish">
+    <PropertyGroup>
+      <VSTestRuntimeConfigPath>$(PublishDir)/vstest.console.runtimeconfig.json</VSTestRuntimeConfigPath>
+      <ReplacementPattern>"version": ".*"</ReplacementPattern>
```

Comment:
Didn't seem to. When I tried on a local build, the runtime.config file got updated successfully to the CLI's runtime version.